src/pybind/mgr/dashboard/rest_client.py

```diff
@@ -0,0 +1,509 @@
+# -*- coding: utf-8 -*-
+"""
+ *   Copyright (c) 2017 SUSE LLC
+ *
+ *  openATTIC is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; version 2.
+ *
+ *  This package is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+"""
+from __future__ import absolute_import
+
+from .tools import build_url
+import inspect
+import itertools
+import re
+import requests
+from requests.exceptions import ConnectionError, InvalidURL
+from . import logger
+
+try:
+    from requests.packages.urllib3.exceptions import SSLError
+except ImportError:
+    from urllib3.exceptions import SSLError
+
+
+class RequestException(Exception):
```

Comment:
Just curious: is there a reason why this and the following exception weren't moved to the newly created `exceptions.py` as well?